src/HealthcareApis/HealthcareApis/HealthcareApisService.csproj

```diff
@@ -0,0 +1,39 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <PsModuleName>HealthcareApisService</PsModuleName>
+  </PropertyGroup>
+
+  <Import Project="$(MSBuildThisFileDirectory)..\..\Az.props" />
+
+  <PropertyGroup>
+    <RootNamespace>Microsoft.Azure.Commands.HealthcareApisService</RootNamespace>
+    <AssemblyName>Microsoft.Azure.PowerShell.Cmdlets.HealthcareApisService</AssemblyName>
+  </PropertyGroup>
```

Comment:
Remove this `<PropertyGroup>` entirely. We have these names set automatically in the `Az.props` file.